elements/pfe-cta/demo/index.html

```diff
@@ -134,7 +121,7 @@ <h3 slot="pfe-card--header"><code>pfe-color="base"</code></h3>
           </div>
           <div class="event-registry" slot="pfe-card--footer"></div>
         </pfe-card>
-        <pfe-card pfe-color="darker" pfe-size="small">
+        <pfe-card pfe-color="darkest" pfe-size="small">
```

Comment:
Same as above - I don't think we should remove this quick visual test case from the page.